frontend-v2/src/app/interest/generator/generator-form.tsx

```diff
@@ -0,0 +1,264 @@
+    queryKey: [API_PATH.CHAPTER_LIST],
+    queryFn: apiClient.getChapterList,
+  })
+
+  const form = useForm<FormValues>({
+    defaultValues: {
+      name: '',
+      title: '',
+      description: '',
+      chapterId: '',
+      activismInterests: false,
+      referralApplyName: '',
+      referralFriends: false,
+      referralApply: false,
+      referralOutlet: false,
+    },
+  })
+
+  const [output, setOutput] = useState('')
+
+  const onSubmit = (data: FormValues) => {
+    const params = new URLSearchParams()
+    params.append('name', data.name)
+    params.append('title', data.title)
+    params.append('description', data.description)
+    params.append('chapterId', data.chapterId)
+    params.append('showInterests', data.activismInterests.toString())
+    params.append(
+      'referralApply',
+      data.referralApplyName === '' ? 'null' : data.referralApplyName,
+    )
+    params.append('showReferralFriends', data.referralFriends.toString())
+    params.append('showReferralApply', data.referralApply.toString())
+    params.append('showReferralOutlet', data.referralOutlet.toString())
+
+    const url = `https://adb.dxe.io/interest?${params.toString()}`
+    setOutput(url)
+  }
+
+  return (
+    <div className="w-full max-w-2xl mx-auto p-8 bg-white rounded shadow">
+      <Form {...form}>
+        <form onSubmit={form.handleSubmit(onSubmit)} className="space-y-6">
+          <FormField
+            control={form.control}
+            name="name"
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>Form name (visible in ADB)</FormLabel>
+                <FormControl>
+                  <Input type="text" placeholder="Name" {...field} />
+                </FormControl>
+              </FormItem>
+            )}
+          />
+
+          <FormField
+            control={form.control}
+            name="title"
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>Form title</FormLabel>
+                <FormControl>
+                  <Input type="text" placeholder="Title" {...field} />
+                </FormControl>
+              </FormItem>
+            )}
+          />
+
+          <FormField
+            control={form.control}
+            name="description"
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>Form description</FormLabel>
+                <FormControl>
+                  <Textarea placeholder="Description" {...field} />
+                </FormControl>
+              </FormItem>
+            )}
+          />
+
+          <FormField
+            control={form.control}
+            name="chapterId"
+            rules={{ required: true }}
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>
+                  Chapter <span className="text-red-500">*</span>
+                </FormLabel>
+
+                <Select
+                  key={field.value}
```

Comment:
shouldn't need this. it's just a workaround for root cause of the issue.
```suggestion
                  key={field.value}
```